packages/Button/Button.md

```diff
@@ -23,10 +27,61 @@ Specify the `variant` to create a button for secondary actions.
 <Button variant="secondary">Find out more</Button>
 ```
 
+### NEW Button variants and ranks
+
+New Buttons and their associated colours are meant for different purposes. Choose different variants according to their context and rank.
+
+#### Ranks
+
+By default, new Button variants will be displayed in the `Common` `rank`. Use Common `rank` for medium to low emphasis.
+
+Use `Main` rank for the most emphasis and main action on a page or in a form.
+
+#### StandardCommon + StandardMain
+
+Used mainly for marketing pages for commerce actions where the direct/immediate action is to purchase. It can also be used for login or registration as these will lead a user to pay their bill or add services.
+
+Examples: TELUS.com, TELUS International, My TELUS
+
+```jsx
+<Button variant="standard">Standard Common</Button>
+```
+
+```jsx
+<Button variant="standard" rank="main">
+  Standard Main
+</Button>
+```
```

Comment:
if main is indeed the default, shouldn't it come first in documentation?

Reply:
it is not.